CHANGELOG/README.md

```diff
@@ -0,0 +1,6 @@
+# Changelogs
+
+* [Current preview changelog](preview.md)
```

Comment:
Why Preview instead of 7.0?

Reply:
So it can stay the same, but either way will work.  My plan was that we create 7.0 when we GA.

Reply:
I see, I'm ok either way.